src/PowerShell/Help/Microsoft.WinGet.Client/Get-WinGetSettings.md

```diff
@@ -0,0 +1,64 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+online version:
+schema: 2.0.0
+---
+
+# Get-WinGetSettings
```

Comment:
[Plural nouns are a big no-no for cmdlet naming](https://learn.microsoft.com/en-us/powershell/scripting/learn/ps101/09-functions?view=powershell-7.4#naming). You can check for precedent on your own machine with `Get-Command -Verb Get -Noun *Setting*`. On mine, `Microsoft.WinGet.Client` is the only offender.

This seems like something that would have been discussed before, when the PowerShell module first gathered feedback. Maybe I don't remember or missed it back then, sorry for just bringing it up now. There's more plural `*Settings` cmdlets below as well. This should probably be its own, non-docs issue too eh?

Reply:
Related to:

- https://github.com/microsoft/winget-cli/issues/4680

Reply:
Thanks so much for finding that, subscribed and bookmarked. Glad it's already tracked, this is resolved then.

Reply:
@jantari it's been a long time coming. Most of the core engineers for WinGet are C++ engineers. We're (including myself as the product owner) all learning about the "Right" way to do PowerShell. I'm happy to learn, and feedback from customers is the most important feedback.

Most of the PowerShell related issues have the https://github.com/microsoft/winget-cli/labels/PowerShell label. Feel free to create more issues as you see fit. That's the best way for us to get the list of things to address. It's going to take time of course, but the more I can open up to the public, the better things should get. As soon as we get past a couple of reviews with the PowerShell team, I'll get these files merged in so we can start the work on the build pipeline to get the help included in the next release.